extensions/adobe/experience/mobile-lifecycle-details.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/mobile-lifecycle-details",
```

Comment:
Please use the following ID pattern to define new MIXINS 
"https://ns.adobe.com/mixins/......."

For this case, it should be
"https://ns.adobe.com/mixins/mobile-lifecycle-details"